ibis-server/tests/routers/v2/connector/test_trino.py

```diff
@@ -97,7 +115,7 @@ def test_query(trino: TrinoContainer):
         "1996-01-02",
         "1_370",
         "2024-01-01 23:59:59.000000",
-        "2024-01-01 23:59:59.000000 UTC",
+        "2024-01-01 23:59:59.000000",
```

Comment:
We didn't change the MDL and input data. Why does this value change? 🤔

Reply:
We changed the expression before, but we did not run CI for trino and did not change the excepted (https://github.com/Canner/wren-engine/pull/782)